templates/InstantSearch.js/.template.js

```diff
@@ -0,0 +1,25 @@
+const build = require('../../packages/tasks/common/build');
+const clean = require('../../packages/tasks/common/clean');
+const install = require('../../packages/tasks/node/install');
+const teardown = require('../../packages/tasks/node/teardown');
+
+module.exports = {
+  libraryName: 'instantsearch.js',
+  tasks: {
+    setup: () => Promise.resolve(),
```

Comment:
Maybe setup could be optional? Maybe other steps could be too?

Reply:
True!